lib/etna/auth.rb

```diff
@@ -77,6 +77,7 @@ def approve_user(request)
 
       begin
         payload, header = application.sign.jwt_decode(token)
+        payload['token'] = token
```

Comment:
Are we sure the payload will never contain the parameter 'token' in the future? Can we instead pass the token as an optional second argument to Etna::User.new?